### DISCUSSION TITLE
$auth.loggedIn state working as expected on local but misbehaving on production

### DISCUSSION THREAD

**Discussion by joshtom** [nuxt-community/auth-module#1619] [Support Requests]
I have a simple flow
- index page
- auth page
- dashboard

I have been able to connect all this page successfully with nuxt auth. But my main issue lies within the index page, where I am trying to conditionally render the avatar image once the user is logged in. However, It works perfectly on local, but didn't work on production.. During the process of debuggin I found out that, the ```$auth.loggedIn``` state is always false.

Below is my code snippet
```
<div class="landingPage__header--authLink" v-if="isLoggedIn">
        <NuxtLink to="/home" class="a-login">
          <a-avatar icon="user" />
        </NuxtLink>
      </div>

      <div class="landingPage__header--authLink" v-else>
        <NuxtLink to="/login" class="a-login"> Log in </NuxtLink>
        <NuxtLink
          to="/register"
          class="landingPage__header--authLink-btn sm-hide"
        >
          Create Account
        </NuxtLink>
      </div>
```

But the funny behaviour is, It was returning true on the dashboard where I'm getting user details and populating dynamically. Any help would be appreciated!

**Comment by productdevbook** [comment]
Have you got any solution? I have the same problem ?

**Comment by joshtom** [comment]
> Have you got any solution? I have the same problem ?

Yeah, I did. But I can't say it's a permanent solution though..
-----> Solution

What I did was check the  state from the localStorage.

Something  like this
```
checkUserLoggedIn() {
      return this.$auth.$storage.getState("loggedIn");
    },
```

**Comment by productdevbook** [comment]
@joshtom https://auth.nuxtjs.org/guide/middleware 
`middleware: 'auth'`
used ? some problem

**Comment by joshtom** [comment]
> @joshtom https://auth.nuxtjs.org/guide/middleware
> `middleware: 'auth'`
> used ? some problem

i don't seem to get this. Can you please rephrase what you meant?

**Comment by productdevbook** [comment]
@joshtom your write custom axios setting ? my axios setting fixed. used now $auth.loggedIn
